tutor/plugins/openedx.py

```diff
@@ -17,7 +17,9 @@ def _migrate_obsolete_nightly_root(root: str) -> None:
 
     REMOVE-ME-AFTER-v20: migrate this code to the sumac upgrade commands.
     """
-    if __version_suffix__ == "main":
+
+    # Run it for old nightly only
+    if __version_suffix__ == "":
```

Comment:
Can I suggest running only in main? `if __version_suffix__ != "main": ...`

Reply:
Sure.

Reply:
updated